packages/whook-swagger-ui/src/index.ts

```diff
@@ -102,7 +102,8 @@ window.onload = function() {
         ? `, {"name":"Private API","url":"${
             publicSwaggerURL +
             '?access_token=' +
-            encodeURIComponent(DEV_ACCESS_TOKEN)
+            encodeURIComponent(DEV_ACCESS_TOKEN) +
+            '?displayOperationId=true'
```

Comment:
Should probably be a & instead. See [URL format](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL#parameters)

Reply:
fixed